app/controllers/users/invitations_controller.rb

```diff
@@ -3,20 +3,27 @@
 module Users
```

Comment:
This spec doesn't exist currently but should create invitations_controller_spec.rb and tests these methods extensively.

Along with testing the working cases, you should also test invalid input as well!

Reply:
Thanks for addressing this!